modules/migration-adding-replication-repository-to-cam.adoc

```diff
@@ -6,11 +6,11 @@
 [id="migration-adding-replication-repository-to-cam_{context}"]
 = Adding a replication repository to the {mtc-short} web console
 
-You can add an object storage bucket as a replication repository to the {mtc-full} ({mtc-short}) web console.
+You can add an object storage as a replication repository to the {mtc-full} ({mtc-short}) web console.
 
 .Prerequisites
 
-* You must configure an object storage bucket for migrating the data.
+* You must configure an object storage for migrating the data.
```

Comment:
@apinnick  I think it would be better if you change it as `You must configure an object storage as replication repository.`